test/governance/Governor.test.js

```diff
@@ -198,31 +198,35 @@ describe('Governor', function () {
       });
 
       describe('vote with signature', function () {
```

Comment:
what about adding stress test edge cases (e.g. signature replay on multiple proposals) or extend the extension to support batch signatures per user?